File: Application/Services/Operations/Outsorced/Dtos/ElectronicRepairDto.cs
using System;
using Application.Services.Operations.Authentication.Dtos;
using Application.Services.Operations.Main.Companies.Dtos;
using Application.Services.Operations.Main.Customers.Dtos;
using Application.Services.Operations.Main.Partners.Dtos;
using Domain.Entities.Authentication;
using Domain.Entities.Main;
using Domain.Entities.Main.Companies;
using Domain.Entities.Main.Customers;
using Domain.Entities.Outsourced.Enums;

namespace Application.Services.Operations.Outsourced.Dtos
{
    public class ElectronicRepairDto
    {
        public int Id { get; set; }
        public int CompanyId { get; set; }
        public CompanyDto Company { get; set; }
        public int UserId { get; set; }
        public MyUserDto User { get; set; }
        public string Item { get; set; }
        public DateTime EntryDate { get; set; }
        public string Description { get; set; }
        public string Problem { get; set; }
        public string UserEquipament { get; set; }
        public string PasswordEquipament { get; set; }
        public Decimal Price { get; set; }
        public int ServiceProviderId { get; set; }
        public PartnerDto ServiceProvider { get; set; }
        public int CustomerId { get; set; }
        public CustomerDto Customer { get; set; }
        public string SolutionApplied { get; set; }
        public StatusServiceEletronicReparEnum Status { get; set; }

    }
}

File: Views/src/components/out-sourced/eletronic-repair/dto/electronic-repair-dto.ts
import { MyUser } from "src/components/authentication/dto/my-user";

import { CustomerDto } from "src/components/main/customer/components/commons-components/dtos/customer-dto";
import { PartnerDto } from "src/components/main/partner/commons-components/dtos/partner-dto";
import { CompanyDto } from "src/shared/entities-dtos/company-dto";
import { StatusServiceElectronicRepairEnumDto } from "./enums/status-service-electronic-repair-enum-dto";

export class ElectronicRepairDto
    {
        id:number;
        companyId:number;
        company:CompanyDto;
        userId:number;
        user:MyUser;
        item:string;
        entryDate:Date;
        description:string;
        problem:string;
        userEquipament:string;
        passwordEquipament:string;
        price:number;
        serviceProviderId:number;
        serviceProvider:PartnerDto;
        customerId:number;
        customer:CustomerDto;
        solutionApplied:string;
        status:StatusServiceElectronicRepairEnumDto;

    }
